vos/test/scripts/vospace-storage-inventory-client-atest.tcsh

```diff
@@ -33,34 +33,20 @@ else
         echo "Using ${TMPDIR} for temporary files"
 endif
 
-set SITE_1_RESOURCE_ID = "ivo://cadc.nrc.ca/minoc"
+set SITE_1_RESOURCE_ID = "ivo://cadc.nrc.ca/alpha-site-ceph/minoc"
```

Comment:
Use the `alpha-site-ceph-4k` as it's the new hotness.

Reply:
Not used with this changes but I will update it for the next task - byte transfers with local storage sites.